src/components/NcAppSidebarTab/NcAppSidebarTab.vue

```diff
@@ -68,12 +70,19 @@ export default {
 	],
 
 	computed: {
-		// TODO: implement a better way to force pass a prop fromm Sidebar
 		isActive() {
-			return this.$parent.activeTab === this.id
+			return this.activeTab === this.id
```

Comment:
:+1:

Reply:
I just realized that this doesn't work in the Tasks app. I have no idea why, though.

Reply:
@ShGKme I think it would be a good idea to revert this change for now. We can still use it when we figure out why it's broken currently. For vue3 it works fine, though.

Reply:
It could be a problem with getter is old Vue is used. Updated with classic getter instead of `computed`.